clang/test/SemaCXX/warn-unsafe-buffer-usage.cpp

```diff
@@ -61,6 +61,7 @@ void testArraySubscripts(int *p, int **pp) {
       );
 
     int a[10];          // expected-warning{{'a' is an unsafe buffer that does not perform bounds checks}}
+                        // expected-note@-1{{change type of 'a' to 'std::array' to harden it}}
```

Comment:
We'll probably have a broader discussion about wording but before I forget: "to harden it" may be misleading because we don't know/check whether library hardening is actually turned on.

Reply:
I am fine with broader discussion later but don't want to block the PR on it.

Reply:
I think it's important to make sure it's not actively misleading, even if we plan to figure out the perfect phrasing later.  Maybe `to incapsulate it for hardening purposes`?

Reply:
I guess what we really want to say is "change the type of 'foo' to std::array so when you build the code with hardened libc++ all subscript operations on 'foo' will be automatically bounds-safe" but that is too long and too complex for this particular communication channel.

How about:
"to enable hardening"
"to get it hardened"
"to designate it for hardening"
"to register it for hardening"

Reply:
"so it can benefit from libc++ hardening"?